packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/index.md

```diff
@@ -56,10 +73,12 @@ To create your app integration in Okta using the CLI:
 
 1. Create the app integration by running:
 
-   ```
+   ```bash
    okta apps create native
    ```
 
+   > **Tip**: If Okta CLI returns an error "Your Okta Org is missing a feature required to use the Okta CLI: API Access Management," you are not using an Okta developer account. To resolve this, see [Set up Okta](#set-up-okta).
```

Comment:
"an error" > "the error"
"you are" > "you're"

Reply:
I always miss a contraction. Thanks.